core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -21,65 +20,63 @@ import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.unit.sp
 import com.iteneum.designsystem.R
 import com.iteneum.designsystem.components.phonenumbertext.mobileNumberFilter
-import com.iteneum.designsystem.theme.Drab
-import com.iteneum.designsystem.theme.MintJulep
 
 /**
  * This function creates a password OutlinedTextField
  * @param modifier Set component modifier
  * @param value Current password value
- * @param errorMessage Set the error message to be displayed when an error occurs
- * @param showError This parameter determines whether the error is displayed or not
+ * @param isValid Validate if text is valid
+ * @param supportTextError This parameter determines whether the error is displayed or not
  * @param onPasswordChange Returns value typed
  *
  * @author Jose G. Rivera
+ * @modifiedBy Jesus Lopez
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun LpOutlinedTextFieldPassword(
-    modifier: Modifier = Modifier,
-    value: String = "",
-    errorMessage: String = "",
-    showError: Boolean = false,
+    modifier: Modifier,
+    value: String,
+    isValid: Boolean,
+    supportTextError: String,
     onPasswordChange: (String) -> Unit
 ) {
+    var textFieldValue by remember { mutableStateOf(TextFieldValue("")) }
```

Comment:
Please be more descriptive for `textFieldValue`,  something to help you how to name them is to consider "what does this field do?" for example in this case I can think something like `passwordFieldValue`